src/components/Table/Table.stories.tsx

```diff
@@ -211,14 +211,35 @@ const rows: Row[] = [
 
 const Template: Story<TableProps> = (args) => {
     const [selectedRows, setSelectedRows] = useState<(string | number)[]>([]);
+    const [sortedRows, setSortedRows] = useState<Row[]>(rows);
+
+    const onSortChange = (key: Key, direction?: SortDirection) => {
+        const sortRows = () => {
+            const clonedRows = [...sortedRows];
+
+            clonedRows.sort((a, b) => {
+                const keyA = a.cells[key].sortId;
+                const keyB = b.cells[key].sortId;
+
+                if (direction !== SortDirection.Descending) {
+                    return keyA < keyB ? -1 : 1;
+                } else {
+                    return keyA < keyB ? 1 : -1;
+                }
+            });
+            setSortedRows(clonedRows);
+        };
+        sortRows();
+    };
 
     return (
         <Table
             {...args}
             columns={columns}
-            rows={rows}
+            rows={sortedRows}
             selectedRowIds={selectedRows}
             onSelectionChange={(ids) => setSelectedRows(ids || [])}
+            onSortChange={onSortChange}
```

Comment:
Should we add the `onSortChange` to the `TemplateWithSearch` story as well?

Reply:
Yup 👌